examples/key_bindings.rs

```diff
@@ -31,18 +31,15 @@ fn setup(
 ) {
     // plane
     commands.spawn((PbrBundle {
-        mesh: meshes.add(Mesh::from(shape::Plane {
-            size: 5.0,
-            ..Default::default()
-        })),
-        material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
+        mesh: meshes.add(Plane3d::new(Vec3::Y)),
```

Comment:
Default Meshable Impl for Plane3d creates a plane with half_size = 1.0,
I think this is fine as it is, but if you want I can set half_size to 2.5 so it's the same as it's now.
I just don't like the way it looks in code.
```
    // plane
    let mut plane = Plane3d::new(Vec3::Y).mesh();
    plane.half_size = Vec2::new(2.5, 2.5);
    commands.spawn((PbrBundle {
        mesh: meshes.add(plane),
        material: materials.add(Color::rgb(0.3, 0.5, 0.3)),
        ..Default::default()
    },));
```

With default half_size = 1.0, it looks like this:
![image](https://github.com/sburris0/bevy_flycam/assets/12927583/4c885700-9974-4daf-9006-6f48ac2197ba)

Also it's relevant for all examples with plane

Reply:
okay nvm, it can be done much easier, I fixed it